browser/android/brave_sync_worker.cc

```diff
@@ -210,28 +196,16 @@ bool BraveSyncWorker::IsFirstSetupComplete(
 bool BraveSyncWorker::ResetSync(
```

Comment:
this function should become `void` as it always returns  `true` now

Reply:
addressed in https://github.com/brave/brave-core/pull/6565/commits/73593a36b3f8013f983fe89691875c1ae00d88e0